lib/common/client/middleware/response/facility_parser.rb

```diff
@@ -29,7 +31,7 @@ def build_facility_attributes(location, mapping)
             facility.merge!(make_address_mappings(location, mapping))
             facility.merge!(make_benefits_mappings(location, mapping, facility['unique_id'])) if mapping['benefits']
             facility.merge!(make_hours_mappings(location, mapping))
-            facility.merge!(make_service_mappings(location, mapping)) if mapping['services']
+            facility.merge!(make_service_mappings(location, mapping, facility['unique_id'])) if mapping['services']
```

Comment:
I know you didn't introduce it and maybe it's just me, but I keep forgetting that `make_service_mappings` is only for health services. What do you think of renaming `make_service_mappings` -> `make_health_service_mappings` and `make_benefits_mappings` -> `make_benefits_service_mappings`? And/or we could rename the service mappings themselves (e.g. `mapping['services']` -> `mapping['health_services']`), but I think changing the methods would be enough.